docs/fundamentals/jina-ai-cloud/login.md

```diff
@@ -0,0 +1,27 @@
+(login)=
+# Authentication
+
+To manage your resources in Jina AI Cloud using DocArray, you need to authenticate to Jina AI Cloud.
+Jina AI Cloud offers several ways to login. Read more about [Login & Token Management in Jina AI Cloud](https://docs.jina.ai/jina-ai-cloud/login/).
+DocArray also offers convenience methods to login/logout using Python API.
+
+## Login
+To login using Python API, use the method {meth}`~docarray.helper.login`:
+```python
+from docarray import login
+
+login()
+```
+The method {meth}`~docarray.helper.login` is interactive, meaning that it will prompt you to login in browser. Non-interactive login options are 
```

Comment:
```suggestion
The {meth}`~docarray.helper.login` method is interactive, meaning that it will prompt you to log in using a browser. Non-interactive login options are 
```